shuup/xtheme/engine.py

```diff
@@ -66,6 +66,10 @@ class XthemeEnvironment(Environment):
 
     template_class = XthemeTemplate
 
+    def __init__(self, *args, **kwargs):
+        super(XthemeEnvironment, self).__init__(*args, **kwargs)
+        self.theme = get_current_theme()
```

Comment:
Are there any potential issues with storing the theme object here? Since it won't be changing, I don't think it will be an issue, but I'm not sure